app/javascript/packages/document-capture/context/upload.tsx

```diff
@@ -0,0 +1,196 @@
+import { createContext } from 'react';
+import { useObjectMemo } from '@18f/identity-react-hooks';
+import type { ReactNode } from 'react';
+import defaultUpload from '../services/upload';
+
+const UploadContext = createContext({
+  upload: defaultUpload,
+  getStatus: () => Promise.reject() as () => Promise<UploadSuccessResponse>,
```

Comment:
Looks like TypeScript isn't as flexible when converting this.

We might have to do a bit more work to convince it that the default value is the type we're expecting:

```suggestion
  getStatus: () => Promise.resolve({} as UploadSuccessResponse),
```